script.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable class-methods-use-this */
+(() => {
+  class BookCatlog {
```

Comment:
Make a separate file for this class and import the file here.

Reply:
done